nsqd/http.go

```diff
@@ -52,6 +52,7 @@ func newHTTPServer(ctx *context, tlsEnabled bool, tlsRequired bool) *httpServer
 	router.Handle("POST", "/pub", http_api.Decorate(s.doPUB, http_api.V1))
 	router.Handle("POST", "/mpub", http_api.Decorate(s.doMPUB, http_api.V1))
 	router.Handle("GET", "/stats", http_api.Decorate(s.doStats, log, http_api.V1))
+	router.Handle("GET", "/producer_stats", http_api.Decorate(s.doProducerStats, log, http_api.V1))
```

Comment:
I'm still _strongly_ in favor of keeping this all in `/stats`, perhaps with a flag to enable.